### DISCUSSION TITLE
I can't save body's background-color.

### DISCUSSION THREAD

**Discussion by souppot** [GrapesJS/grapesjs#4553] [Q&A]
Hi,
I have a question about a decoration of body tag.
Even if I specify the background-color of the body tag and save it, it will not be reproduced next time.

I save html and css by next methods (get by next methods and post to the server(php)).
`var htmldata = editor.getHtml();`
`var cssdata = editor.getCss();`

And when I want to load, set like this.
`editor.setComponents(htmldata);`
`editor.setStyle(cssdata );`

Maybe it is because grapes.js ignore the id of the body tag when loaded. How can I resolve this?

**Comment by artf** [comment]
I don't see any issue with the latest version, if you're able to provide a reproducible demo, it might help.

**Reply by souppot** [reply]
Thank you so much for your reply.
I will show you some sample.

(1) I created the html with grapes.js as in the attached capture.
[grapes.pdf](https://github.com/artf/grapesjs/files/9524180/grapes.pdf)

I got html & css data by getHtml()/getCss() as below.
css
`* { box-sizing: border-box; } body {margin: 0;}#ii6l{background-color:#f0e987;}#i6df{padding:10px;}`
html
`<body id="ii6l"><div id="i6df">TEST STRING</div></body>`

(2) I tried to preset them in grapes.js editor. The attached txt file is a sample of html & js. As a result, "TEST STRING" was printed, but the background-color was ignored.
[test.txt](https://github.com/artf/grapesjs/files/9524203/test.txt)
[output.pdf](https://github.com/artf/grapesjs/files/9524265/output.pdf)

Please help me.

**Reply by artf** [reply]
Oh sure, you can't use `editor.setComponents` with a body tag inside (it's removed when HTML is parsed).
You're basically trying to load a new project without relying on the storageManager. For your case, you can make use of [`editor.loadProjectData`](https://grapesjs.com/docs/api/editor.html#loadprojectdata)

**Reply by souppot** [reply]
I'm sorry for the late reply. But it worked! Thank you so much for your kindness.
